app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,64 @@
+class InvitesController < ApplicationController
+  skip_before_action :authorize_invite_access
+
+  before_action :check_invite_required, only: [:required, :show]
+
+  load_and_authorize_resource :only => [:new, :create]
+
+  layout :resolve_layout
+  def required
+  end
+
+  def new
+    @invite = Invite.new
+    @form = InviteForm.new(@invite)
+  end
+
+  def create
+    @form = InviteForm.new(Invite.new)
+    if @form.validate(invite_params)
+      @invite = @form.model
+      InviteCreator.new(@invite.email).perform!
+
+      flash[:alert] = "Invite has been sent to #{@invite.email}"
+      redirect_to new_invite_url
+    else
+      render :new
+    end
+  end
+
+  def show
+    @invite = Invite.find_by!(:code => params[:id])
```

Comment:
Perhaps we could standardise, as code field is confusing given usage of `token` elsewhere.